src/binaryen-c.cpp

```diff
@@ -3570,6 +3570,10 @@ void BinaryenSetLowMemoryUnused(int on) {
   globalPassOptions.lowMemoryUnused = on != 0;
 }
 
+int BinaryenGetFastMath(void) { return globalPassOptions.fastMath; }
+
+void BinaryenSetFastMath(int value) { globalPassOptions.fastMath = value != 0; }
```

Comment:
These also need to be declared in `binaryen-c.h` using `BINARYEN_API`.

Reply:
right! Thanks